OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -188,20 +188,25 @@ void LiveFunctionsDataView::OnContextMenu(
     }
   } else if (a_Action == MENU_ACTION_JUMP_TO_FIRST) {
     CHECK(a_ItemIndices.size() == 1);
-    JumpToNext(GetFunction(a_ItemIndices[0]),
-               std::numeric_limits<TickType>::min());
+    auto function_address = GetFunction(a_ItemIndices[0]).GetVirtualAddress();
+    TextBox* first_box = GCurrentTimeGraph->GetNext(
+        function_address, std::numeric_limits<TickType>::min());
+    GCurrentTimeGraph->JumpToBox(first_box);
   } else if (a_Action == MENU_ACTION_JUMP_TO_LAST) {
     CHECK(a_ItemIndices.size() == 1);
-    JumpToPrevious(GetFunction(a_ItemIndices[0]),
-                   std::numeric_limits<TickType>::max());
+    auto function_address = GetFunction(a_ItemIndices[0]).GetVirtualAddress();
+    TextBox* last_box = GCurrentTimeGraph->GetPrevious(
+        function_address, std::numeric_limits<TickType>::max());
+    GCurrentTimeGraph->JumpToBox(last_box);
   } else if (a_Action == MENU_ACTION_JUMP_TO_MIN) {
     CHECK(a_ItemIndices.size() == 1);
-    auto [min_box, _] = GetMinMax(GetFunction(a_ItemIndices[0]));
-    JumpToBox(min_box);
+    Function& function = GetFunction(a_ItemIndices[0]);
+    auto [min_box, _] = GetMinMax(function);
```

Comment:
I've commented something similar on b/160851767. We shouldn't recompute min/max every time this action is triggered. TimerChain (and BlockChain) data structures are never reallocated, so you can just store pointers to the TextBox that you need to jump to (that is, store one ptr for first/min/max/last per function). Ideally this should be done when computing the stats in the first place.

Reply:
This is much less a problem than I initially thought (the issue was that the window was not redrawn). For very long captures, this is likely still a problem, but probably ok to leave as-is for now.

Reply:
Yes, this is the plan. Ok to leave as-is for now and change it later (in fact, it has pros and cons).